### PR TITLE
README typo/grammar fixes.

### PR DESCRIPTION
**Pull request by bmfay**
Proposed fixes.  Mostly typos, two grammar changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,21 +12,21 @@ response status, rendering validation errors, and inserting changesets.
 
 ## Installation
 
-If [available in Hex](https://hex.pm/docs/publish), the package can be installed as:
+If [available in Hex](https://hex.pm/docs/publish), the package can be installed by:
 
-  1. Add ja_resource to your list of dependencies in `mix.exs`:
+  1. Adding ja_resource to your list of dependencies in `mix.exs`:
```
Comment (bmfay) [inline README.md]: So, "the package can be installed as: 1) Add .. 2) Ensure" doesn't work grammatically. This change does work grammatically, but if you want it to be explicitly directive in tense (keeping it as Add / Ensure) then I'd suggest something like:

"the package can be installed with the following steps: 1) Add.. 2) Ensure.."

File: README.md [hunk 2]
```diff
@@ -147,9 +147,9 @@ end
 
 ### 'Handle' Actions
 
-Every action not excluded defines a default `handle_` variant receives
-pre-processed data and is expected to return an Ecto query or record. All of 
-the handle calls may also return a conn (including the result of a render 
+Every action not excluded defines a default `handle_` variant which receives
+pre-processed data and is expected to return an Ecto query or record. All of
+the handle calls may also return a conn (including the result of a render
```
Comment (bmfay) [inline README.md]: Had basically [subject] [verb] [object] [verb] [object] here.  If I understood your explanation in flowdock this makes it work grammatically and retains the meaning you intended.

**Comment by bmfay** [comment]
@alanpeabody

**Comment by alanpeabody** [comment]
Awesome, thanks!

**Pull request merged**